### ISSUE TITLE
Match Changelog Icon Size with New Design

### ISSUE THREAD

**Issue by SudharakaP** [opencollective/opencollective#4473]
We need to match the changelog icon notification title with the new figma designs. 

https://www.figma.com/file/ZQBMWhnGGtRWeIZknFW1eP/%5BOC-Design%5D-Production-Ready-%E2%9C%85?node-id=12889%3A182814

Currently it looks like;

![image](https://user-images.githubusercontent.com/12435965/126374780-f8f63a54-1191-4ef9-9d40-d89cc8837bf3.png)

It should be updated to;

![image](https://user-images.githubusercontent.com/12435965/126374828-867475ef-5503-4a13-bdc1-6434dea27684.png)

Basically the outer circle should be made larger (40 x 40 px). 😄 

And thanks @Memo-Es for pointing this out. 😸
